package.json

```diff
@@ -44,6 +46,7 @@
     "eslint-plugin-react": "^7.18.3",
     "file-loader": "^5.1.0",
     "jest": "^25.1.0",
+    "jest-enzyme": "^7.1.2",
```

Comment:
We don't need any of the jest or enzyme stuff here if we don't have tests to run but I guess it doesn't really hurt.